interfaces/builtin/desktop.go

```diff
@@ -110,6 +110,20 @@ dbus (receive)
     member={ActionInvoked,NotificationClosed,NotificationReplied}
     peer=(label=unconfined),
 
+dbus (send)
```

Comment:
Can you please add a comment here about why this is required (ie for the Inhibited property) and then also a link to some upstream KDE documentation which documents this property?